Airline_Web_API/Airline_Web_API/Controllers/ApiController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Airline_Web_API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ApiController : Controller
+    {
+        public override void OnActionExecuting(ActionExecutingContext context)
+        {
+            if (!ModelState.IsValid)
+            {
+                var errors = ModelState
+                    .SelectMany(x => x.Value.Errors)
+                    .ToList();
+
+                context.Result = new BadRequestObjectResult(errors);
+            }
+        }
+    }
+}
```

Comment:
That's what I have by now. I'm not sure about this decision as I don't know what is better in my case: delete ApiController attribute or add my realization to this method. I'll make final version in some days according to prons and cons of every solution.